notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py

```diff
@@ -210,10 +210,6 @@
 table = pd.Series(counts, index=genes).sort_values(ascending=False)
 # EPY: END code
 
-# EPY: START code
-table.head()
```

Comment:
un-nuke this

Reply:
It is present directly below, this is repetitive.